src/components/ImportTokenButton.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react'
```

Comment:
What is the advantage of having this custom component over generic RN [Button](https://reactnative.dev/docs/button)?